examples/test-app/views/HelloWorld.html

```diff
@@ -1,5 +1,14 @@
-{% from "patterns/header.njk" import header %}
+{% extends "layouts/two_thirds.html" %}
```

Comment:
It's not clear to me where "layouts/two_thirds.html" lives, it would be better to be explicit such as "look-and-feel/layouts/two_thirds.html"